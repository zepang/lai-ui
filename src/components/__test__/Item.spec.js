import Vue from 'vue'
import Alert from './Alert.vue'
import { mount, shallowMount } from '@vue/test-utils'

// const instance = new Vue({
//   render: (h) => {
//     return h(Alert, {})
//   }
// })

// const vm = instance.$mount()

describe('Item.vue', () => {
  test('sanity test', () => {
    // expect(wrapper.vm.$el.textContent).toContain('Alert')  
    const wrapper = shallowMount(Alert, {sync: false})
    debugger
    expect(wrapper.text()).toContain('Alert')
    return
  })
  test('has the expected html structure', () => {
    expect(wrapper.vm.$el).toMatchSnapshot()
  })
  test('assert test', () => {
    expect(true).toBe(true)
  })
  
})