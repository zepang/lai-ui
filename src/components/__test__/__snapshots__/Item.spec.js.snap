// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item.vue has the expected html structure 1`] = `
<div>
  Alert
</div>
`;
